src/app/components/AccountInfo/index.tsx

```diff
@@ -74,11 +71,11 @@ class AccountInfo extends React.Component<Props, State> {
             <div className="AccountInfo-top-info">
               <div className="AccountInfo-top-info-alias">{account.alias}</div>
               <div className="AccountInfo-top-info-balance">
-                <Unit value={account.totalBalance} showFiat />
+                <Unit value={account.totalBalance} chain={chain} showFiat />
```

Comment:
`<Unit>` is already a connected component, and should be running the selector itself, instead of adding it to every instance of it.